Library/Homebrew/test/cask/system_command_spec.rb

```diff
@@ -1,4 +1,20 @@
 describe Hbc::SystemCommand, :cask do
+  describe "#initialize" do
+    describe "given some environment variables" do
+      subject do
```

Comment:
Use curly braces for `subject`.

Reply:
Will do once we have addressed the trivial expectation thing.